cardano_node_tests/tests/tests_conway/test_conway_extras.py

```diff
@@ -0,0 +1,925 @@
+        drep2 = get_custom_drep(
+            name_template=f"custom_drep_2_{temp_template}",
+            cluster_manager=cluster_manager,
+            cluster_obj=cluster,
+            payment_addr=payment_addr,
+            caching_key=key2,
+        )
+
+        # Create stake address registration cert
+        reg_cert = cluster.g_stake_address.gen_stake_addr_registration_cert(
+            addr_name=f"{temp_template}_addr0",
+            deposit_amt=deposit_amt,
+            stake_vkey_file=pool_user.stake.vkey_file,
+        )
+
+        reqc.cli087.start(url=helpers.get_vcs_link())
+        # Create vote delegation cert for drep 1
+        deleg_cert_1 = cluster.g_stake_address.gen_vote_delegation_cert(
+            addr_name=f"{temp_template}_addr1",
+            stake_vkey_file=pool_user.stake.vkey_file,
+            drep_key_hash=drep1.drep_id,
+            always_abstain=False,
+            always_no_confidence=False,
+        )
+
+        # Create vote delegation cert for drep 2
+        deleg_cert_2 = cluster.g_stake_address.gen_vote_delegation_cert(
+            addr_name=f"{temp_template}_addr2",
+            stake_vkey_file=pool_user.stake.vkey_file,
+            drep_key_hash=drep2.drep_id,
+            always_abstain=False,
+            always_no_confidence=False,
+        )
+
+        # Submit two vote delegation certificate at once
+        tx_files = clusterlib.TxFiles(
+            certificate_files=[reg_cert, deleg_cert_2, deleg_cert_1],
+            signing_key_files=[payment_addr.skey_file, pool_user.stake.skey_file],
+        )
+
+        # Make sure we have enough time to finish the registration/delegation in one epoch
+        clusterlib_utils.wait_for_epoch_interval(
+            cluster_obj=cluster, start=1, stop=common.EPOCH_STOP_SEC_LEDGER_STATE
+        )
+
+        clusterlib_utils.build_and_submit_tx(
+            cluster_obj=cluster,
+            name_template=temp_template,
+            src_address=payment_addr.address,
+            use_build_cmd=True,
+            tx_files=tx_files,
+            deposit=deposit_amt,
+        )
+        stake_addr_info = cluster.g_query.get_stake_addr_info(pool_user.stake.address)
+
+        assert stake_addr_info.vote_delegation == governance_utils.get_drep_cred_name(
+            drep_id=drep1.drep_id
+        ), "Votes are NOT delegated to the correct DRep 1 placed at last of certificates list."
+        reqc.cli087.success()
+
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.parametrize("drep", ("always_abstain", "always_no_confidence", "custom"))
+    @pytest.mark.testnets
+    @pytest.mark.smoke
+    def test_no_delegation_without_stake_registration(
+        self,
+        cluster: clusterlib.ClusterLib,
+        payment_addr: clusterlib.AddressRecord,
+        pool_user: clusterlib.PoolUser,
+        custom_drep: governance_utils.DRepRegistration,
+        drep: str,
+    ):
+        """Test Change delegation to different dreps.
```

Comment:
Wrong description

Reply:
Thank you for the feedback. It has been addressed and moved to test_drep.py file.